src/visualizers/widgets/PlotlyGraph/PlotlyGraphWidget.js

```diff
@@ -23,8 +25,9 @@ define([
         this.$el.css('overflow', 'auto');
         this.$el.addClass(WIDGET_CLASS);
         this.nodes = {};
-        this.plotlyJSON = null;
+        this.plotlyJSONS = null;
```

Comment:
Actually, it's not clear to me that it needs to be saved on `this`. If possible, it would be preferable to avoid this extra statefulness and simply pass the the plotly JSON(s) to the functions that require them.

Reply:
Also, `this.nodes`, `this.layout`, `this.created`, and `this._container` appear to be unused (or unnecessary).